ql/src/test/queries/clientpositive/ptf_tnk.q

```diff
@@ -0,0 +1,22 @@
+CREATE EXTERNAL TABLE t1(
```

Comment:
Would it be possible to also load some data and verify that the results of the query are correct?

Reply:
Added data to the test.